serialization.md

```diff
@@ -1,7 +1,7 @@
 # OCI Image Serialization
 
 An *Image* is an ordered collection of root filesystem changes and the corresponding execution parameters for use within a container runtime.
-This specification outlines the format of these filesystem changes and corresponding parameters and describes how to create and use them for use with a container runtime and execution tool.
+This specification outlines the JSON format describing images for use with a container runtime and execution tool and its relationship to filesystem change sets, described in [Layers](layer.md).
 
 ## Terminology
```

Comment:
On Thu, Aug 25, 2016 at 07:44:39AM -0700, Stephen Day wrote:
> >  ## Terminology
> 
> This is terminology that is specific to the context of this format.

“Layer” is more general than this format, since it's now covered in a
separate file (layer.md).  “Tag” and “Repository” seem like they're
waiting for some future naming spec, and don't seem to be specific to
this file's serialization spec.  If we want to keep a terminology
section for the serialization spec, we should at least restrict it to
serialization-specific terms.